lib/tasks/mastodon.rake

```diff
@@ -479,16 +497,16 @@ namespace :mastodon do
             prompt.say 'Running `RAILS_ENV=production rails assets:precompile` ...'
             prompt.say "\n\n"
 
-            if system(env.transform_values(&:to_s).merge({ 'RAILS_ENV' => 'production' }), 'rails assets:precompile')
-              prompt.say 'Done!'
-            else
+            if command.run!(:rails, 'assets:precompile', env: env.transform_values(&:to_s).merge({ 'RAILS_ENV' => 'production' })).failure?
               prompt.error 'That failed! Maybe you need swap space?'
+            else
+              prompt.say 'Done!'
             end
           end
         end
 
         prompt.say "\n"
-        prompt.ok 'All done! You can now power on the Mastodon server 🐘'
+        prompt.ok 'All configured!'
```

Comment:
Have made this change as ideally we don't want them to start their server until they've imported a blocklist